src/controllers/issue-controller.ts

```diff
@@ -11,7 +11,8 @@ export class IssueController {
                 "label": activity.name,
                 "description": "",
                 "detail": "",
-                "fullIssue": activity
+                "fullIssue": this.issue,
+                "id": activity.id
```

Comment:
So, if you would keep activity inside fullIssue key, then this key wouldn't be needed (since `id` would be available via fullIssue key)